crates/aptos/src/common/types.rs

```diff
@@ -738,6 +738,40 @@ pub trait ParsePrivateKey {
     }
 }
 
+#[derive(Debug, Default, Parser)]
+pub struct HardwareWalletOptions {
+    /// Derivation Path of your account in hardware wallet
+    ///
+    /// e.g format - [m/44'/637'/0'/0'/0]
+    /// Make sure your wallet is unblocked and have Aptos opened
+    #[clap(long)]
+    pub derivation_path: Option<String>,
+
+    /// Index of your account in hardware wallet
+    ///
+    /// This is the simpler version of derivation path e.g format - [0]
+    /// we will translate this index into [m/44'/637'/0'/0'/0]
+    #[clap(long)]
+    pub derivation_index: Option<String>,
```

Comment:
TBH, I feel like these names are too long. Open for better naming suggestion

Reply:
These shouldn't be used particularly often, so I say it's fine.  I think the real question is whether these apply to hardware wallets directly that are not Ledger.  Would be cool to see eventually something like Keystone or otherwise.

Reply:
Keystone?

I did some research, looks like most of the hardware wallet does uses derivation path. But yeah, lets keep these for now for general hardware wallet. And we can made adjustment when we support more than 1 HD wallet

Reply:
They pretty much all use derivation paths and secret phrases so I think we should be good.

Don't worry about keystone for now :)